src/components/Stats/__tests__/Stats-test.tsx

```diff
@@ -14,7 +16,12 @@ describe('Stats', () => {
 
   it('should render <Template data= />', () => {
     const wrapper = mount(
-      <Stats {...getProps()} templateProps={{ templates: defaultTemplates }} />
+      (
+        <Stats
+          {...getProps()}
+          templateProps={{ templates: defaultTemplates }}
+        />
+      ) as ReactElementLike
```

Comment:
Casting it as `ReactElementLike` because we return an `Element`, not sure that this is the best way to do it though.

Reply:
We can convert the test to the Testing Library. We want to move away from Enzyme.

Reply:
Since we want to migrate later, it's good that you first made the existing tests passing. I don't have a big opinion on whether it stays like this now or gets migrated